docs/source/notebooks/AR.ipynb

```diff
@@ -16,7 +16,7 @@
   },
```

Comment:
<p>Same comments as for ar2</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/pymc-devs/pymc3/pull/3899/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='14'/>